src/canillita.erl

```diff
@@ -29,13 +35,15 @@ start_phase(create_schema, _StartType, []) ->
     {error, {Node, {already_exists, Node}}} -> ok
   end,
   {ok, _} = application:ensure_all_started(mnesia),
-  sumo:create_schema();
+  ok = sumo:create_schema(),
+  pg2:create(canillita_listeners);
```

Comment:
This should be a different `start_phase` and it should include the piece from the comment above.
The phase should be called `start_event_managment` or something like that